app/frontend/src/views/Admin.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+  <BaseSecure :admin="true">
+    <v-container>
+      <transition name="component-fade" mode="out-in">
+        <router-view />
+      </transition>
+    </v-container>
+  </BaseSecure>
+</template>
+
+<script>
+import { mapGetters } from 'vuex';
+
+import admin from '@/store/modules/admin.js';
+
+export default {
+  name: 'Admin',
+  computed: {
+    ...mapGetters('auth', ['isAdmin']),
+  },
+  created() {
+    if (this.$store.hasModule('admin')) {
+      this.$store.unregisterModule('admin');
+    }
+    if (this.isAdmin) {
+      this.$store.registerModule('admin', admin);
+    }
+  },
+};
+</script>
+
+<style scoped>
+.component-fade-enter-active,
```

Comment:
We might need to make these transition things get pushed up to top-level scss - it's starting to show up in multiple places now.

Reply:
Yeah will look, have a todo with table styles in the same way